### DISCUSSION TITLE
Does containerd.io default support Mount propagation

### DISCUSSION THREAD

**Discussion by ming12713** [longhorn/longhorn#6356] [General]
Hi, In the best practices, it is mentioned that 'Mount propagation' needs to be configured. In my own cluster environment, I am using containerd, not Docker. Does containerd.io support Mount propagation by default?"
![Screenshot from 2023-07-20 11-10-39](https://github.com/longhorn/longhorn/assets/5196113/b00def09-6b85-4142-8fba-2676fb7b897a)
**my cluster env** 
kubernetes: v1.26.6
containerd.io: v1.7.2

**Comment by mantissahz** [comment]
`containerd` should have support MountPropagation after https://github.com/containerd/cri/issues/185 and https://github.com/containerd/cri/pull/218.
And `MountPropagation` is a GA feature and unconditionally enabled after Kubernetes 1.14 released

**Reply by ming12713** [reply]
nice ,thanks
